flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/expressions/PlannerExpressionConverter.scala

```diff
@@ -79,12 +79,6 @@ class PlannerExpressionConverter private extends ApiExpressionVisitor[PlannerExp
         val windowReference = translateWindowReference(children.head)
         return WindowEnd(windowReference)
 
-      case THROW_EXCEPTION =>
```

Comment:
Interesting.  Seems reasonable to remove this.  

Also, seems unrelated.

Do we need the THROW_EXCEPTION function?  Or can it be removed in general?

Reply:
We still have it. I guess you are mislead that it is not listed in `BuiltinFunctionDefinitions` but `InternalFunctionDefinitions`